tests/namespacing/ignore-invalid/transforms/apache_parser.toml

```
@@ -1,4 +1,7 @@
 # Structure and parse the data
-inputs = ["apache_logs"]
-type = "regex_parser" # fast/powerful regex
-patterns = ['^(?P<host>[w.]+) - (?P<user>[w]+) (?P<bytes_in>[d]+) [(?P<timestamp>.*)] "(?P<method>[w]+) (?P<path>.*)" (?P<status>[d]+) (?P<bytes_out>[d]+)$']
+inputs        = ["apache_logs"]
+type          = "remap"
+drop_on_error = false
+source        = '''
+. |= parse_apache_log!(string!(.message), "common")
+	'''
```

```suggestion
'''
```

Maybe? TOML formatting is weird